### PR TITLE
[FIX] Displaying profile managers for Coverage model in Public share

### PR DESCRIPTION
**Pull request by harini-venkataraman**
## What
Invoking exposed API for displaying Profile managers in Coverage model
...

## Why
Public share needs to have whitelisted API invoked for public user.
...

## How
Fixes by invoking the public API for fetching adapters and profile manager.
...

## Can this PR break any existing features. If yes, please list possible items. If no, please explain why. (PS: Admins do not merge the PR without this section filled)
No, this fixes the invocation for Public share API.
...

## Database Migrations
Not applicable
...

## Env Config
Not applicable
...

## Relevant Docs
Not applicable
...

## Related Issues or PRs

-

## Dependencies Versions
Not applicable
...

## Notes on Testing
Can be tested with execution of Public prompt share and prompting the coverage model.
-

## Screenshots
image

## Checklist

I have read and understood the [Contribution Guidelines]().

### CODE REVIEW

**Review by Deepak-Kesavan — APPROVED** [review]

**Comment by Deepak-Kesavan** [comment]
@harini-venkataraman The frontend build is failing. Please check and fix it.

**Review by tahierhussain — APPROVED** [review]

File: frontend/src/components/custom-tools/output-for-doc-modal/OutputForDocModal.jsx [hunk 1]
```diff
@@ -172,12 +175,14 @@ function OutputForDocModal({
   };
 
   const getAdapterInfo = () => {
-    axiosPrivate
-      .get(`/api/v1/unstract/${sessionDetails.orgId}/adapter/?adapter_type=LLM`)
-      .then((res) => {
-        const adapterList = res.data;
-        setAdapterData(getLLMModelNamesForProfiles(llmProfiles, adapterList));
-      });
+    let url = `/api/v1/unstract/${sessionDetails.orgId}/adapter/?adapter_type=LLM`;
```
Comment (vishnuszipstack) [inline frontend/src/components/custom-tools/output-for-doc-modal/OutputForDocModal.jsx:178]: @harini-venkataraman "/api/v1/unstract/${sessionDetails.orgId}" can be defined in a path variable and reuse in repeating places. 
REF:- https://github.com/Zipstack/unstract/blob/1fb2d4e849e8ec0134050fb43db01597b98f4f52/frontend/src/components/deployments/api-deployment/api-deployments-service.js#L9

**Review by vishnuszipstack — CHANGES_REQUESTED** [review]
Left a single comment. Please resolve this.

File: frontend/src/components/custom-tools/output-for-doc-modal/OutputForDocModal.jsx (hunk 1 above)
Comment (harini-venkataraman) [reply frontend/src/components/custom-tools/output-for-doc-modal/OutputForDocModal.jsx:178]: @vishnuszipstack This change looks like a FE refactor, which we can scope it in a separate PR and refactor for all files. 
CC : @tahierhussain

**Review by vishnuszipstack — APPROVED** [review]

**Pull request merged**